### PR TITLE
Fix transition for cancelled light bulb in integration tests

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs [hunk 1]
```diff
@@ -61,6 +61,8 @@ public static async Task<IEnumerable<SuggestedActionSet>> WaitForItemsAsync(Test
 
                 if (e.Status == QuerySuggestedActionCompletionStatus.Completed)
                     tcs.SetResult(e.ActionSets.ToList());
+                else if (e.Status == QuerySuggestedActionCompletionStatus.Canceled)
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs:64]: do we know what is canceling hte lightbulb?

Comment (CyrusNajmabadi) [reply src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs:64]: effectively, if the lightbulb is triggered, and the user doesn't do something to cancel it (whcih i would not expect tests to do), then i would not expect this to ever get hi.

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs:64]: sometimes the editor just dismisses the light bulb, even if we don't request it.

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs:64]: ➡️ 23d48db should fix the unexpected dismissal

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs:64]: ➡️ Now moved to #60048

**Review by CyrusNajmabadi — APPROVED** [review]

**Pull request merged**